ethcore/builtin/src/lib.rs

```diff
@@ -245,6 +245,13 @@ impl Builtin {
 	pub fn is_active(&self, at: u64) -> bool {
 		self.pricer.range(0..=at).last().is_some()
 	}
+
+	/// EIP 1352: Restricted address range for precompiles/system contracts
+	#[inline]
+	pub fn in_precompiles_address_range(addr : &Address) -> bool {
+		let limit = Address::from_low_u64_ne(0xffff);
```

Comment:
this is not correct
```
>> let limit = ethereum_types::Address::from_low_u64_ne(0xffff);
>> limit
0x000000000000000000000000ffff000000000000
```
and we could use a `const` here

Reply:
should be `Address::from_low_u64_be(0xffff)` right?

Reply:
> should be `Address::from_low_u64_be(0xffff)` right?

either that or
```rust
const LIMIT: Address = Address([0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0xff,0xff]);
```

Reply:
fixed in fixed in b34b753